src/main/java/com/puppycrawl/tools/checkstyle/checks/AbstractTypeAwareCheck.java

```diff
@@ -41,7 +41,6 @@
  * @author Oliver Burn
  * @deprecated Checkstyle is not type aware tool and all Checks derived from this
  *     class are potentially unstable.
- * @noinspection DeprecatedIsStillUsed
```

Comment:
I am not sure how TC is configured. This configuration might not be related to TC build.

`WARNING` would be reported as warning in TC as well I guess.